soc/arm/st_stm32/stm32wb/Kconfig.defconfig.series

```diff
@@ -12,11 +12,9 @@ config SOC_SERIES
 
 if PM
 config PM_DEVICE
-	bool
 	default y
 
 config STM32_LPTIM_TIMER
-	bool
```

Comment:
Nitpick: A "+ minor fixes" in the commit message for this would have been nice

Reply:
added in the commit msg